### DISCUSSION TITLE
已经删除的计划任务还会执行

### DISCUSSION THREAD

**Discussion by Calcifer97** [1Panel-dev/1Panel#6010] [Q&A]
### 联系方式

zhanglor@163.com

### 1Panel 版本

v1.10.13-lts

### 问题描述

现在看只有两条计划任务
![image](https://github.com/user-attachments/assets/e6db792e-c28d-470e-a12c-903a1cc322f8)
但是日志中有还有一条多余的，备份OPT这个计划任务是之前有的，已经被删除的
![image](https://github.com/user-attachments/assets/6ecdb8aa-ba0e-4ecc-ac27-105f952450a4)

### 重现步骤

先创建一个备份OPT到本地磁盘，然后修改这个计划任务上传到OSS，然后删除这个计划任务。

### 期待的正确结果

删除的任务不应该出现

**Comment by Calcifer97** [comment]
我现在应该怎么去数据库删除这条记录？

**Comment by ssongliu** [comment]
感谢反馈，本地环境没有复现该问题，删除该计划任务的时候有报错或者超时吗？

可以通过重启 1panel 服务停掉这个计划任务。
